src/components/FormLabel/FormLabel.tsx

```diff
@@ -68,21 +69,33 @@ class FormLabel extends React.Component<Props, State> {
     const {
       className,
       children,
+      inline,
       for: htmlFor,
       id: idProp,
       ...rest
     } = this.props
 
-    const componentClassName = classNames('c-FormLabel', className)
+    const componentClassName = classNames(
+      'c-FormLabel',
+      inline && 'c-FormLabel--inline',
```

Comment:
I've actually started to drop the BEM convention, and stuck with prefix based modifier names like `is-inline`.
May reasons is because it's simpler (less verbose). And since these classNames are controlled and scoped to the component, the chances if it conflicting its probably the same as the BEM version.

@tinkertrain What you think ? :D

Reply:
I think is great, never been a big fan of BEM's super long class names. And I think think they are as useful in component and css-in-js world :)